lib/client.js

```diff
@@ -13,8 +13,26 @@ var hawk        = require('hawk');
 var url         = require('url');
 var crypto      = require('crypto');
 var slugid      = require('slugid');
+var http        = require('http');
+var https       = require('https');
 var Promise     = require('promise');
 
+// Default options for our http/https global agents...
+var AGENT_OPTIONS = {
+  maxSockets: 5,
+  keepAlive: true
+}
```

Comment:
Missing `;` :)

Also can we add a `exports.configAgent` that will allow updates to `AGENT_OPTIONS` and update `DEFAULT_AGENTS`.

It wouldn't and shouldn't affect instances of client classes already created, but those created after the call...
Granted we can also choose just do this later if it'll ever be needed.

I think it makes sense for servers to use `25` connections or more.. .But limit `maxFreeSockets` to maybe 10...